docker/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+killbill:
```

Comment:
This is great, but the only thing is that now we have [multiple](https://github.com/killbill/killbill-cloud/blob/master/docker/compose/docker-compose.kb.yml) `docker-compose.yml` to maintain (update versions,...)